### ISSUE TITLE
Screen reader not announcing name of edit field announcing only place holder value in Issue Reporter

### ISSUE THREAD

**Issue by kisantia** [microsoft/vscode#194075]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.82.2
- OS Version: Windows

Steps to Reproduce:
1. Open vscode and navigate to Issue Reporter
2. Start screenreader (NVDA or Narrator)
3. Navigate through dropdowns and input boxes and see that only values/placeholders are announced, but not the names next to these input fields.

Expected Result:
Screen reader should announce name of combo box and value containing it also.

Speech-input user will not be able to navigate to Steps to Reproduce text field as its name is not as its visual.
![issueReporterAriaLabelsVscode](https://github.com/microsoft/vscode/assets/31145923/ac185a8d-d9b4-485e-be3f-7f4a5e39f938)

original issues created in ADS: https://github.com/microsoft/azuredatastudio/issues/24219, https://github.com/microsoft/azuredatastudio/issues/24218

**Comment by cpichuka** [comment]
@TylerLeonhardt , Do you have ETA for this bug fix?

**Comment by TylerLeonhardt** [comment]
No ETA at this time.
